main.go

```diff
@@ -23,6 +24,10 @@ func main() {
 	serverConf := server.Config{
 		HTTPListenAddr: conf.Server.Address,
 		HTTPListenPort: conf.Server.Port,
+		HTTPMiddleware: []middleware.Interface{
+			gateway.NewAuthentication(&conf),
+		},
+		UnAuthorizedHTTPListenPort: 6666,
```

Comment:
I could not decide how to put this into the configuration file. Because of that, I hard-coded in main.go file for now. I would like to you hear your thoughts on how should we add unauth information to the config file.